example/dotnet/test/UnitTest1.cs

```diff
@@ -0,0 +1,12 @@
+using Xunit;
+
+namespace MyApp.Tests;
+
+public class UnitTest1
+{
+    [Fact]
+    public void Test1()
+    {
+
+    }
+}
```

Comment:
As mentioned above, we could also consider fleshing this out and adding in a dotnet test to the integration suite.